src/tests/dictionaryConcepts/container/DictionaryConcepts.test.jsx

```diff
@@ -99,43 +102,6 @@ describe('Test suite for dictionary concepts components', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
-  it('should render a loader', () => {
-    const props = {
```

Comment:
Is this change part of the ticket?

Reply:
@dkayiwa yes, this is part of the ticket. There were two loaders that are shown when a search is done, one was through a component that's imported into the `DictionaryConcepts` file and the other was on `ReactTable` component. The team asked me to have consistency with the search done on the Bulk Concepts page and therefore there was no need for that loader. So I took it out.